src/themes/default/components/core/ProductGalleryZoomCarousel.vue

```diff
@@ -2,10 +2,13 @@
   <div class="media-zoom-carousel">
     <div class="media-zoom-carousel__container row flex">
       <ul class="media-zoom-carousel__thumbs m0 p0">
-        <li class="media-zoom-carousel__thumb" v-for="(images, key) in gallery" :key="images.src">
-          <span class="bg-cl-secondary block">
-            <img :src="images.src" ref="images" @click="navigate(key)" :alt="productName | htmlDecode">
-          </span>
+        <li class="media-zoom-carousel__thumb" v-for="(images, index) in gallery" :key="images.src">
+          <product-image
+            :images="images"
+            :alt="productName | htmlDecode"
+            :hidden="hideImageAtIndex === index"
+            @click.native="navigate(key)"
```

Comment:
also child component should emit `click` event so we don't have to use native here